### PR TITLE
fix some type of openstack_volumes.go

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
fix some type of openstack_volumes.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/assign @dims 
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/cc @FengyunPan2 
thanks

File: pkg/cloudprovider/providers/openstack/openstack_volumes.go [hunk 1]
```diff
@@ -553,14 +552,10 @@ func (os *OpenStack) getDevicePathFromInstanceMetadata(volumeID string) string {
 func (os *OpenStack) GetDevicePath(volumeID string) string {
 	devicePath := os.GetDevicePathBySerialID(volumeID)
 
-	if devicePath == "" {
-		devicePath = os.getDevicePathFromInstanceMetadata(volumeID)
-	}
-
 	if devicePath == "" {
 		glog.Warningf("Failed to find device for the volumeID: %q", volumeID)
+		devicePath = os.getDevicePathFromInstanceMetadata(volumeID)
```
Comment (NickrenREN) [inline pkg/cloudprovider/providers/openstack/openstack_volumes.go]: We want to get device path from `id` and if necessary from `metadata service`, if no one succeeds, then log error message. I don't think we need to do this change.

**Review by NickrenREN — CHANGES_REQUESTED** [review]

**Comment by dims** [comment]
/close 

per last comment from @NickrenREN

**Pull request closed without merging**